distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java

```diff
@@ -78,6 +78,8 @@ public void beforeAllianceTest() throws Exception {
             getServiceManager().startFeature(true, "nitf-render-plugin");
             getCatalogBundle().waitForCatalogProvider();
             configureSecurityStsClient();
+            configureRestForGuest();
+            getSecurityPolicy().waitForGuestAuthReady(REST_PATH.getUrl() + "?_wadl");
```

Comment:
If we're always waiting for the same path, perhaps it should be pushed down into the `waitForGuestAuthReady` method.